src/server.ts

```diff
@@ -56,19 +54,62 @@ export class Server {
     }
   }
 
+  /**
+   * Fetch a minimal set of current info about a Stellar account.
+   *
+   * Needed to get the current sequence number for the account so you can build
+   * a successful transaction with {@link TransactionBuilder}.
+   *
+   * @example
+   * server.getAccount("GBZC6Y2Y7Q3ZQ2Y4QZJ2XZ3Z5YXZ6Z7Z2Y4QZJ2XZ3Z5YXZ6Z7Z2Y4").then(account => {
+   *   console.log("sequence:", account.sequence);
+   * });
+   *
+   * @param {string} address - The public address of the account to load.
+   * @returns {Promise<SorobanRpc.GetAccountResponse>} Returns a promise to the {@link SorobanRpc.GetAccountResponse} object with populated sequence number.
+   */
   public async getAccount(
     address: string,
   ): Promise<SorobanRpc.GetAccountResponse> {
     return await jsonrpc.post(this.serverURL.toString(), "getAccount", address);
   }
 
+  /**
+   * General node health check.
+   *
+   * @example
+   * server.getHealth().then(health => {
+   *   console.log("status:", health.status);
+   * });
+   *
+   * @returns {Promise<SorobanRpc.GetHealthResponse>} Returns a promise to the {@link SorobanRpc.GetHealthResponse} object with the status of the server ("healthy").
+   */
   public async getHealth(): Promise<SorobanRpc.GetHealthResponse> {
     return await jsonrpc.post<SorobanRpc.GetHealthResponse>(
       this.serverURL.toString(),
       "getHealth",
     );
   }
 
+  /**
+   * Reads the current value of contract data ledger entries directly.
+   *
+   * @example
+   * const contractId = "0000000000000000000000000000000000000000000000000000000000000001";
+   * const key = xdr.ScVal.scvSymbol("counter");
+   * server.getContractData(contractId, key).then(data => {
+   *   console.log("value:", data.xdr);
+   *   console.log("lastModified:", data.lastModifiedLedgerSeq);
+   *   console.log("latestLedger:", data.latestLedger);
+   * });
+   *
+   * Allows you to directly inspect the current state of a contract. This is a backup way to access your contract data which may not be available via events or simulateTransaction.
+   *
+   * @deprecated Use {@link Server#getLedgerEntry} instead.
```

Comment:
Can we make sure there's an item somewhere to drop this? It's too early to have long-lived deprecations :joy:

Reply:
https://github.com/stellar/soroban-tools/issues/316